cwf/gateway/services/uesim/servicers/uesim_hssless.go

```diff
@@ -0,0 +1,133 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package servicers
+
+import (
+	cwfprotos "magma/cwf/cloud/go/protos"
+	"magma/feg/gateway/services/aaa/session_manager"
+	lte_protos "magma/lte/cloud/go/protos"
+	"magma/orc8r/cloud/go/blobstore"
+	"magma/orc8r/lib/go/protos"
+	"strings"
+
+	"github.com/golang/glog"
+	"golang.org/x/net/context"
+)
+
+const (
+	imsiPrefix    = "IMSI"
+	defaultUeIp   = "192.168.88.88"
+	defaultApn    = "wifi"
+	KiloBytes     = 1024
+	defaultMSISDN = "5100001234"
+)
+
+// UESimServerHssLess tracks all the UEs being simulated.
+type UESimServerHssLess struct {
+	store blobstore.BlobStorageFactory
+	cfg   *UESimConfig
+}
+
```

Comment:
Since all the major RPCs (Authenticate, Disconnect, GenTraffic) behave differently for hssless as compared to the normal uesim,  i did not encapsulate UeSimServer inside UESimServerHssLess.  More details  in further comments.